app/services/support_interface/change_application_choice_course_option.rb

```diff
@@ -0,0 +1,47 @@
+module SupportInterface
+  class ChangeApplicationChoiceCourseOption
+    VALID_STATES = ApplicationStateChange::DECISION_PENDING_STATUSES
+
+    attr_reader :application_choice, :provider_id, :course_code, :study_mode, :site_code, :audit_comment
+
+    def initialize(application_choice_id:,
+                   provider_id:,
+                   course_code:,
+                   study_mode:,
+                   site_code:,
```

Comment:
Given that (so far) providers seem to be providing us with the site name only, is there a way we make it easier to ascertain the correct site code? This was the most time consuming part of the process IMO. Or perhaps this can be best dealt with when we add the functionality to the Support UI ( e.g having a course and site drop down)

Reply:
I did consider that but its easy for someone to mispell the full site name and I think as part of the process we should be going to the course page and retrieving the code for some extra verification. 

Maybe this is something that we can include in the documentation for now as it might not be straightforward for someone who is not as familiar with the provider course pages.

In the make offer wizard we display the available course and then filter down to study mode and location, so I think that would be how we eventually want to be dealing with that in support, and essentially amending the service to accept the selected course option rather than the combination of fields that retrieve it.